stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -261,6 +269,33 @@ function __init__()
                  """
         end
 
+        ### Check that the Common and Method Julia structs are consistent with their C counterparts
+        method_offsets = Vector{Csize_t}(undef,16)
```

Comment:
Is there a reason this is length 16 and not 15 ? Same with the `common_offsets` array which should store 107 elements I think?

Reply:
Yes, that's wrong. Fixed it, sorry about that.